crates/debugger_ui/src/debugger_panel.rs

```diff
@@ -272,6 +272,24 @@ impl DebugPanel {
         })
     }
 
+    fn handle_continued_event(
+        this: &mut Self,
+        client_id: &DebugAdapterClientId,
+        event: &ContinuedEvent,
+        cx: &mut ViewContext<Self>,
+    ) {
+        let all_threads = event.all_threads_continued.unwrap_or(false);
+        let client = this.debug_client_by_id(client_id.clone(), cx);
```

Comment:
I think we can deref here, by adding `* client_id` instead of cloning it.

Reply:
Fixed! Thanks for the feedback